### DISCUSSION TITLE
Question: Grouping of Windows

### DISCUSSION THREAD

**Discussion by nliechti** [nikitabobko/AeroSpace#627] [questions-and-answers] [answered]
I am new to using Aerospace and really like it.

I have one question regarding "force grouping" of windows. I have a lot of Intellij Project open at the same time and used the "Always open windows in Tabs" which seems to be a very broken feature for WM's.
Is there some way or workaround to group all IntelliJ windows in a vertical stack as if it is one window?
I don't seem to be able to move the hole stack around as I would like.

Thanks for the great work 👍

**Comment by nikitabobko** [comment, marked as answer]
> I don't seem to be able to move the hole stack around as I would like.

It's not possible, but I understand your use case. A concrete proposals on how it could work are welcome (e.g. what flags, what commands or maybe a boolean flag inside a layout need to be added). Description of more use cases are welcome too

In i3, it's done via `focus child|parent` #5 But I am not sure that I like the i3 approach because users always need to firstly run `focus parent` before moving a stack of windows around.
